osbs/core.py

```diff
@@ -43,14 +43,15 @@
 logger = logging.getLogger(__name__)
 
 
-def check_response(response):
+def check_response(response, log_level=logging.ERROR):
     if response.status_code not in (httplib.OK, httplib.CREATED):
         if hasattr(response, 'content'):
             content = response.content
         else:
             content = b''.join(response.iter_lines())
 
-        logger.error("[%d] %s", response.status_code, content)
+        log_message = "[%d] %s" % (response.status_code, content)
+        logger.log(log_level, log_message)
```

Comment:
logging module should be the one performing string formatting. Please change it to:
`logger.log(log_level, "[%d] %s", response.status_code, content)`